v2/internal/pkg/clusterresources/clusterresources.go

```diff
@@ -366,8 +393,98 @@ func catalogSourceContentFromTemplate(templateFile, catalogSourceName, image str
 	return obj, nil
 }
 
-func (o *ClusterResourcesGenerator) generateIDMS(mirrorsByCategory []categorizedMirrors) ([]confv1.ImageDigestMirrorSet, error) {
+// FIXME: support generation with template (or equivalent)
```

Comment:
@r4f4 - If this is the fix - then could you remove the FIXME label ?

Reply:
It's not the fix. We don't support using a template yet like with `CatalogSource`: https://github.com/openshift/oc-mirror/blob/main/v2/internal/pkg/clusterresources/clusterresources.go#L203-L205

So this is a TBA and the PM agreed the first iteration doesn't need to support templates.

Reply:
I can remove the comment of course, if you think it's just unnecessary noise.

Reply:
@r4f4 - ok I get the context now, I think leave it in or maybe rename to TODO with the comment

Reply:
Done